src/typescript/services/todoService.ts

```diff
@@ -0,0 +1,32 @@
+import { API_BASE_URL } from '../constants/urls';
+import Todo from '../models/todoModel';
+import ApiService from './apiService';
+
+class TodoService {
+    private api: ApiService;
```

Comment:
Did you consider about inherit?

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!